src/PProf.jl

```diff
@@ -331,7 +331,7 @@ function refresh(; webhost::AbstractString = "localhost",
 end
 
 """
-    pprof_kill()
+    kill()
```

Comment:
Actually, since we don't export it, this should probably be:
```suggestion
    PProf.kill()
```